docs/source/notebooks/Boson_Bunching.ipynb

```diff
@@ -0,0 +1,1076 @@
+       "<path d=\"M2875,125 L3125,125\" stroke=\"darkred\" stroke-width=\"3\" fill=\"none\" />\n",
+       "<path d=\"M2625,175 L3125,175\" stroke=\"darkred\" stroke-width=\"3\" fill=\"none\" />\n",
+       "<path d=\"M2375,225 L3125,225\" stroke=\"darkred\" stroke-width=\"3\" fill=\"none\" />\n",
+       "<path d=\"M2125,275 L3125,275\" stroke=\"darkred\" stroke-width=\"3\" fill=\"none\" />\n",
+       "<path d=\"M1875,325 L3125,325\" stroke=\"darkred\" stroke-width=\"3\" fill=\"none\" />\n",
+       "<path d=\"M3125,25 L3140,25\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,75 L3140,75\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,125 L3140,125\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,175 L3140,175\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,225 L3140,225\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,275 L3140,275\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<path d=\"M3125,325 L3140,325\" stroke-width=\"3\" stroke=\"darkred\" stroke-linejoin=\"miter\" fill=\"none\" />\n",
+       "<text x=\"3150\" y=\"28\" font-size=\"8\" text-anchor=\"end\">0</text>\n",
+       "<text x=\"3150\" y=\"78\" font-size=\"8\" text-anchor=\"end\">1</text>\n",
+       "<text x=\"3150\" y=\"128\" font-size=\"8\" text-anchor=\"end\">2</text>\n",
+       "<text x=\"3150\" y=\"178\" font-size=\"8\" text-anchor=\"end\">3</text>\n",
+       "<text x=\"3150\" y=\"228\" font-size=\"8\" text-anchor=\"end\">4</text>\n",
+       "<text x=\"3150\" y=\"278\" font-size=\"8\" text-anchor=\"end\">5</text>\n",
+       "<text x=\"3150\" y=\"328\" font-size=\"8\" text-anchor=\"end\">6</text>\n",
+       "<text x=\"0\" y=\"28\" font-size=\"8\" text-anchor=\"start\">0</text>\n",
+       "<text x=\"0\" y=\"78\" font-size=\"8\" text-anchor=\"start\">1</text>\n",
+       "<text x=\"0\" y=\"128\" font-size=\"8\" text-anchor=\"start\">2</text>\n",
+       "<text x=\"0\" y=\"178\" font-size=\"8\" text-anchor=\"start\">3</text>\n",
+       "<text x=\"0\" y=\"228\" font-size=\"8\" text-anchor=\"start\">4</text>\n",
+       "<text x=\"0\" y=\"278\" font-size=\"8\" text-anchor=\"start\">5</text>\n",
+       "<text x=\"0\" y=\"328\" font-size=\"8\" text-anchor=\"start\">6</text>\n",
+       "</svg>"
+      ],
+      "text/plain": [
+       "<drawsvg.drawing.Drawing at 0x28028e530>"
+      ]
+     },
+     "execution_count": 17,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# building Mach-Zender Interferometer block of the circuit\n",
+    "mzi = (pcvl.BS() // (0, pcvl.PS(phi=pcvl.Parameter(\"φ_a\")))\n",
```

Comment:
You can actually define this MZI from the catalog:
`pcvl.catalog["mzi phase last"].build_circuit(phi_a=pcvl.Parameter(\"φ_a\"), phi_b=pcvl.Parameter(\"φ_b\"))`